website/README.md

```diff
@@ -118,6 +118,42 @@ The code for the client is located under `src/client/`. The webpack build create
 
 Scripts related to deployment, like the Dockerfile, are under `deploy`. Note: even though the scripts are under `deploy`, you must run them from the **root of the repository**; they are sensitive to `cwd`.
 
+## Query Parameters
+
+You can pass certain parameters in the Snack URL to customize the behavior of the Snack.
+
+Here is a summary of all the parameter options that you can use:
+
+| parameter | default | description | type |
+| -------- | ------- | ----------- | ------ |
+| **dependencies** | | Dependencies to be used by your Snack beyond common dependencies and the ones added in the Snack's `package.json`. [Read more about Snack dependencies](_`string[https://github.com/expo/snack/blob/main/docs/snack-sdk.md#using-dependencies]`_). | [SnackDependency](https://github.com/expo/snack/blob/main/packages/snack-content/src/types.ts#L64) |
+| **description** | | The description of your Snack. | `string` |
+| **files** | | Allows you to inline files to load through a query param, useful for doc maintainers who don't want to maintain twice the code (both markdown and snack, itself). [Read more about files](https://github.com/expo/snack/blob/main/docs/snack-sdk.md#files-dependencies-and-the-state). | [`SnackFiles`](https://github.com/expo/snack/blob/main/packages/snack-content/src/types.ts#L40) |
+| **name** | | The name of your Snack. When creating a new Snack, a random name is assigned unless one is provided via this parameter.  | `string` |
+| **platform** | `web` | The platform on which your Snack should be run in the Device Preview window. | `android`, `ios`, `mydevice`, `web` |
+| **preview** | `true` | Toggle to show the Device Preview. | `boolean` |
+| **sdkVersion** | `44.0.0` | The Expo SDK version that your Snack should use. | `45.0.0`, `44.0.0`, `43.0.0` |
```

Comment:
We likely will forget to update this, so I think it would be better to just link the types 😅 

```suggestion
| **sdkVersion** | _[default SDK](../packages/snack-content/src/defaults.ts#L3)_ | The Expo SDK version that your Snack should use. | [SDKVersion](../packages/snack-content/src/sdks/types.ts#L4) |
```